src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -111,44 +145,52 @@ AppendGrantOnDatabaseStmt(StringInfo buf, GrantStmt *stmt)
 }
 
 
+static bool
+AppendBasicAlterDatabaseOptions(StringInfo buf, DefElem *def, bool
```

Comment:
We usually omit `Append...` function comments, but here you could explain in the comment what are the "Basic" alter database options.

Reply:
Added comment

Reply:
@gurkanindibay Actually that is not what I meant, sorry maybe I wasn't clear enough. I meant to write in the comment what are the basic alter database options, meaning is_template, allow_connections, connection_limit and tablespace.
I suggest adding a comment with these to clarify what we are doing here. As you know, ALTER DATABASE has more options than that which actually fall into different categories https://www.postgresql.org/docs/current/sql-alterdatabase.html

Reply:
Added more details